src/App.svelte

```diff
@@ -143,46 +174,67 @@
         <div id="download-link" class="link-item">
           <span class="material-icons-outlined header-download-icon"
             >file_download</span
           >
-          Download data
+          {$_('navigation.download_data')}
         </div>
         <div class="link-item">
           <a
             class="link-item-anchor"
             href="https://storage.googleapis.com/gcs-public-datasets/COVID-19%20Vaccination%20Search%20Insights%20documentation.pdf"
-            >Documentation</a
+            >{$_('navigation.documentation')}</a
           >
         </div>
       </div>
     </div>
     <div id="header-download-popup" class="header-download-popup">
       <h3 class="header-downlod-popup-title">
-        Covid-19 Vaccination Search Insights
+        {$_('project_title')}
       </h3>
       <p class="header-download-popup-body">
-        In order to download or use the data or insights, you must agree to the
-        Google
-        <a href="https://policies.google.com/terms">Terms of Service</a>.
+        {@html $_('content.download_popup.terms', {values: {
+          toDownloadsUrl: "https://policies.google.com/terms"}})}
       </p>
-      <h4 class="header-download-popup-subtitle">Download dataset</h4>
+      <h4 class="header-download-popup-subtitle">{$_('content.download_popup.download_dataset')}</h4>
       <p class="header-download-popup-link-list">
-        {#each COUNTRY_LIST as countryName}
-          <a
-            class="header-download-popup-link"
-            href={"https://storage.googleapis.com/covid19-open-data/covid19-vaccination-search-insights/" +
-              fetchCountryMetaData(countryName)[0].countryCode + "_vaccination_search_insights.csv"}
-            on:click={(e) => closeDownloadPopup()}
-            ><span class="material-icons-outlined header-download-popup-icon">file_download</span>
-            {countryName}
-          </a>
-        {/each}
+        <a
+          class="header-download-popup-link"
+          href="https://storage.googleapis.com/covid19-open-data/covid19-vaccination-search-insights/CA_vaccination_search_insights.csv"
+          on:click={(e) => closeDownloadPopup()}
+          ><span class="material-icons-outlined header-download-popup-icon"
+            >file_download</span
+          >{$_('content.download_popup.countries.canada')}</a
+        >
+        <a
+          class="header-download-popup-link"
+          href="https://storage.googleapis.com/covid19-open-data/covid19-vaccination-search-insights/GB_vaccination_search_insights.csv"
+          on:click={(e) => closeDownloadPopup()}
+          ><span class="material-icons-outlined header-download-popup-icon"
+            >file_download</span
+          >{$_('content.download_popup.countries.uk')}</a
+        >
+        <a
+          class="header-download-popup-link"
+          href="https://storage.googleapis.com/covid19-open-data/covid19-vaccination-search-insights/GB_vaccination_search_insights.csv"
```

Comment:
Please change the filename here to IE_vaccination_search_insights.csv, as currently it download the UK data for Ireland

Reply:
done.